src/styles-to-add.css

```diff
@@ -15,4 +15,15 @@
 .webtreemap-caption {
   font-size: 10px;
   text-align: center;
-}
\ No newline at end of file
+}
+
+.webtreemap-node:focus {
+  border-color: hsla(215, 100%, 48%, 0.247);
```

Comment:
I noticed that this looks funny for wrapped siblings (some parts of their focus border is behind the previous elements) e.g. selecting prettier in the demo

![image](https://user-images.githubusercontent.com/639216/63546515-d8c3b880-c4de-11e9-86fa-3e8190f0b8c8.png)